.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -61,8 +62,7 @@ data class CIBuildModel (
             Stage("Experimental", "On demand: Run experimental tests",
                     trigger = Trigger.never,
                     runsIndependent = true,
-                    functionalTests = listOf(
-                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java10))))
```

Comment:
So the experimental stage will be empty? It doesn't seem correct.

Reply:
Yes, it will be empty, because there are no more experiments left. At some point we could add Java 11 here.

Reply:
I'm wondering if this will cause issues? Is empty configuration allowed?

Reply:
Ping @donat @oehme